apps/daimo-mobile/src/view/shared/AmountInput.tsx

```diff
@@ -42,8 +46,35 @@ export function AmountChooser({
 }) {
   // Show how much we have available
   const [account] = useAccount();
+  const dispatcher = useContext(DispatcherContext);
+
   if (account == null) return null;
   const dollarStr = getAmountText({ amount: account.lastBalance });
+  const showHelpModal = () =>
+    dispatcher.dispatch({
+      name: "helpModal",
+      title: "Why are there no fees?",
+      content: (
+        <>
+          <TextLight>Daimo uses Base, an Ethereum rollup.</TextLight>
+          <Spacer h={24} />
+          <TextLight>
+            Rollups support near-instant transactions that cost a few cents each
+            by bundling many transactions into a single L1 transaction. They
+            inherit the strong guarantees of Ethereum: like L1, Base is reliable
+            and secure, and works worldwide.
+          </TextLight>
```

Comment:
Second paragraph:

| Daimo transactions are sponsored. This means that your transfers are free.

Reply:
I just replaced second paragraph with this 👆

Sorry if you meant something else 🫠